manifests/m/Mariocube/EasyCSM/1.4/Mariocube.EasyCSM.locale.en-US.yaml

```
@@ -0,0 +1,12 @@
+# Created using wingetcreate 1.2.5.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.4.0.schema.json
+
+PackageIdentifier: Mariocube.EasyCSM
+PackageVersion: 1.4
+PackageLocale: en-US
+Publisher: Mariocube
+PackageName: Easy CSAM

```

```suggestion
PackageName: ECSM
```
Must match whatever is written to registry (shown in Apps&Features or `appwiz.cpl`)